### ISSUE TITLE
On older versions of IDA, the `idaapi.get_member_by_id` function will return a malformed `idaapi.struc_t`.

### ISSUE THREAD

**Issue by arizvisa** [arizvisa/ida-minsc#135]
For some reason, the `idaapi.get_member_by_id` function from IDAPython can return an `idaapi.struc_t` that appears to be malformed. This function is \_supposed\_ to return a tuple composed of `(mptr: idaapi.member_t, fullname: str, sptr: idaapi.struc_t)`. However, in some cases I've been getting an `sptr` that returns a bunk value whenever I try to access its `id` property.

Typically the value it returns is zero, but here's an example where it returns 48.
```
Python>id=0xff000000000003f6L
Python>a,b,c=idaapi.get_member_by_id(id)
Python>c.id
1394703794176
Python>x=c.id
Python>x
48
```

**Comment by arizvisa** [comment]
Tagging this as both a bug and an enhancement, because the fix for this will be a workaround for an IDAPython issue.

**Comment by arizvisa** [comment]
Closing this issue as it was fixed by PR #136.
